pkg/cli/image/manifest/manifest.go

```diff
@@ -437,7 +467,7 @@ func ProcessManifestList(ctx context.Context, srcDigest digest.Digest, srcManife
 			if err != nil {
 				return nil, nil, "", err
 			}
-			klog.Warningf("Chose %s/%s manifest from the manifest list.", t.Manifests[0].Platform.OS, t.Manifests[0].Platform.Architecture)
+			klog.Warningf("Chose %s manifest from the manifest list.", PlatformSpecString(t.Manifests[0].Platform))
```

Comment:
I don't think this is appropriate at all - we sholud be returning a typed error, NOT printing inline warnings from library methods.  No CLI code should ever call into stuff using klog.

Reply:
It slid by before, in a previous PR (that was mine) - I've removed this, makes sense, thanks for clarifying that.